src/common/serializers/event.serializer.ts

```diff
@@ -0,0 +1,82 @@
+import { deserializeDirectoryUser } from '../../directory-sync/serializers';
+import { deserializeConnection } from '../../sso/serializers';
+import {
+  Webhook,
+  WebhookBase,
+  WebhookResponse,
+} from '../../webhooks/interfaces';
+import {
+  deserializeDeletedWebhookDirectory,
+  deserializeUpdatedWebhookDirectoryGroup,
+  deserializeUpdatedWebhookDirectoryUser,
+  deserializeWebhookDirectory,
+  deserializeWebhookDirectoryGroup,
+} from '../../webhooks/serializers';
+
+export const deserializeEvent = (event: WebhookResponse): Webhook => {
```

Comment:
I put this in `common` because it's going to be used by both the Events API and the Webhooks. While the types are now `Webhook` and `WebhookResponse` I'm going to be consolidating the types for both of these under the name `Event`.